.github/workflows/ci.yml

```diff
@@ -643,11 +643,16 @@ jobs:
           sed -n '1,/^$/p' distribution/changelog.txt >> release_notes.txt
           echo "" >> release_notes.txt
 
+          echo "Release created in ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}" >> release_notes.txt
+          echo "" >> release_notes.txt
+
           echo "SHA1 checksums:" >> release_notes.txt
           echo "\`\`\`" >> release_notes.txt
           cat OpenRCT2-${{ needs.build_variables.outputs.name }}-sha1sums.txt >> release_notes.txt
           echo "\`\`\`" >> release_notes.txt
           echo "" >> release_notes.txt
+
+          cat release_notes.txt
```

Comment:
I'm assuming this is for debug only and will go away

Reply:
I'd like it to stay. It makes the job much easier in case something goes wrong at a later stage